tests/framework/constants.go

```diff
@@ -66,3 +66,9 @@ const (
 	// ControlPlaneOnly will collect logs only for control plane processes
 	ControlPlaneOnly CollectLogsType = "controlPlaneOnly"
 )
+
+const (
+	// EnvoyOSMWindowsImage is Envoy Windows image used for testing.
+	// On Windows until Windows Server 2022 is publicly available we have to rely on this testing images.
+	EnvoyOSMWindowsImage = "openservicemesh/envoy-windows-nanoserver:latest"
```

Comment:
Could we pin this a specific version instead of latest to follow tagging best practices?

Reply:
yeah for sure

Reply:
fixed by pinning the image to the sha digest